src/libraries/illink-sharedframework.targets

```diff
@@ -70,6 +58,30 @@
       <ILLinkArgs>$(ILLinkArgs) -r @(_AssembliesToLink->'%(Identity)', ' -r ')</ILLinkArgs>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <ProjectILLinkSuppressionsFile>src\ILLink\ILLink.Suppressions</ProjectILLinkSuppressionsFile>
+      <CoreLibSharedILLinkSuppressionsFilesPrefix>$(CoreLibSharedDir)ILLink\ILLink.Suppressions.Shared</CoreLibSharedILLinkSuppressionsFilesPrefix>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <!-- Include suppression XML files bin-placed in earlier per-library linker run. -->
+      <_SuppressionsXmls Include="$(ILLinkTrimAssemblySuppressionsXmlsDir)*.xml" />
+
+      <!-- Collate CoreLib suppression XML files not bin-placed in earlier per-library linker run. CoreLib doesn't use bin-place logic. -->
+      <_SuppressionsXmls Include="$(CoreLibSharedILLinkSuppressionsFilesPrefix).xml" />
+      <_SuppressionsXmls Condition="'$(TargetOs)' != 'Windows_NT'" Include="$(CoreLibSharedILLinkSuppressionsFilesPrefix).NonWindows.xml" />
```

Comment:
(nit) `TargetOs` => `TargetOS`. I noticed this in `eng\illink.targets` as well. I know MSBuild is case-insensitive, but it is better to be consistent, for easier reading.